serverless.yaml

```diff
@@ -133,6 +133,17 @@ functions:
           maximumRetryAttempts: 3
           startingPosition: LATEST
 
+  ddbToEsDlq:
```

Comment:
What's the trigger here? I'm not seeing the lambda set to run on each SQS queued message. Are we going to do this on a cron schedule? Mainly concerned about the sync fails Friday at 5:01 PM and we fail throughout the weekend until Monday 9AM scenario.

Reply:
Good point. @lwarren01 Do you have any thoughts ?

Reply:
So far, this lambda needs to be triggered manually.

Reply:
That's a good question. Since this queue is being treated as a DLQ, my understanding is that you don't automatically have a process consuming off the queue. It does require some manual intervention, to determine whether it is safe to re-try. What the Lambda function provides is a mechanism to re-process (re-drive) the messages once that is the case. 

We should be monitoring the queue so that we can be alerted to messages arriving on it. Determine what might be the cause, address the issue, then invoke the Lambda to re-try.

Perhaps we need to better understand what might cause the DDBToES Lambda function to fail, resulting in messages appearing on the queue. For example, if it is just congestion, then perhaps we can look at configuring a trigger for this Lambda to re-try at slow rate. I am not sure though.